docs/userguide/packaging/index.md

```diff
@@ -95,8 +99,7 @@ data to be installed and removed without rebuilding ICU. For details, see the
 
 (This section assumes the reader is familiar with ICU version numbers (§) as
 covered in the [Design](../design.md) chapter, and filename conventions for
-libraries in the    
-[ReadMe](http://source.icu-project.org/repos/icu/icu/trunk//readme.html#HowToPackage)
+libraries in the [ReadMe](../../../icu4c/readme.html#HowToPackage)
```

Comment:
I'm wondering if this should be a relative link or an absolute link. Relative link is probably OK because we will want to continue keeping documentation alongside code.

Reply:
right. many other items seemed to have such relative links